themes/delphi/assets/uikit/package.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
Is there a reason all of these uikit files are included in this draft?  If they need to live outside of node_modules, there should be a script to move them into the right place and then .gitignore them rigth?

Reply:
that is the hugo theme 1:1 from the www-main website. In this example it is copied on purpose such that in a different version we can share the theme more easily